normandy/recipes/models.py

```diff
@@ -227,6 +196,10 @@ def update_signature(self):
             self.signature = None
             return
```

Comment:
Why not a `.save()` here? It used to be a .save here when it was do in `update_signatures()`.

Reply:
As I mentioned in my other comment, `Recipe.update_signature` is called during `Recipe.save`. `RecipeQuerySet.update_signatures` was not.

Reply:
Now I see where the save happens, it happens here: https://github.com/mythmon/normandy/blob/0db4bd58b1303bcc2f8be6a45652b0029ec7cb3e/normandy/recipes/management/commands/update_recipe_signatures.py#L39